File: src/main/java/econtroller/controller/NewNodeRequest.java
package econtroller.controller;

import se.sics.kompics.address.Address;
import se.sics.kompics.network.Message;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-20
 *
 */

public class NewNodeRequest extends Message {

	private static final long serialVersionUID = -7505388733307482046L;

	public NewNodeRequest(Address source, Address destination) {
		super(source, destination);
	}

}

File: src/main/java/econtroller/controller/NewNodeRequest.scala
package econtroller.controller

import se.sics.kompics.network.Message
import se.sics.kompics.address.Address

/**
 * @author Amir Moulavi
 * @date 2011-07-13
 *
 */
class NewNodeRequest(source:Address, destination:Address, val numberOfNodes:Int) extends Message(source, destination)